pfetch

```diff
@@ -652,6 +652,11 @@ get_pkgs() {
         (IRIX)
             packages=$((packages - 3))
         ;;
+        # OpenBSD's wc prints whitespace before the number
+        # which needs to be stripped.
+        (OpenBSD)
+            packages="$(echo $packages)"
+        ;;
```

Comment:
what about just packages=$((packages)) ? Then there's no unsafe word-splitting or subshell.

desert$ var='   10    '
desert$ echo "$var"
   10    
desert$ echo "$((var))"
10
desert$

Reply:
ah, didn't know this was a thing. neat.